rel8.cabal

```diff
@@ -46,6 +46,7 @@ library
     -Wno-missing-import-lists -Wno-prepositive-qualified-module
     -Wno-monomorphism-restriction
     -Wno-missing-local-signatures
+    -Wno-missing-kind-signatures
```

Comment:
Is your preference to provide a kind signature only for type constructors?

Reply:
I got into the habit of writing standalone kind signatures when writing Rel8, so almost all our types had them already. The few that didn't, I added signatures for them with this PR. The big exception is the first class families stuff:

```haskell
data TColumn :: K.Context -> Type -> Exp Type
type instance Eval (TColumn f a) = Column f a
```

From the point of view of `-Wmissing-kind-signatures`, `TColumn` has no kind signature. And there's no nice way to give it one without having to repeat yourself. So that's why I turned it off.

I think standalone kind signatures are nice, and they're especially useful for Rel8, but I don't think it's worth being dogmatic about it,

Reply:
Perfect, thanks for clarifying! That all makes sense.